CMakeLists.txt

```diff
@@ -179,7 +179,7 @@ if(MEGA)
 else()
 	if(MSVC)
 		message(FATAL_ERROR "
-It is currently only possible to build with megasource on Windows.
+It is currently only possible to build with megasource or mingw on Windows.

```

Comment:
Could we add a notice or warning while using mingw that it's not officially supported? Since we don't plan to maintain it ourselves, so it might break in the future.

Reply:
Sure that makes sense, adjusted in 88ea145.